finder-app/writer.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <string.h>
+#include <syslog.h>
+
+void print_usage()
+{
+    syslog(LOG_DEBUG, "Total number of arguements should be 2");
+    syslog(LOG_DEBUG, "The order of arguements should be:");
+    syslog(LOG_DEBUG, "  1) File directory path");
+    syslog(LOG_DEBUG, "  2) String to be written in to the specified file directory path");
+}
+
+int main(int argc, char** argv)
+{
+    openlog(NULL, 0, LOG_USER);
+
+    if (argc != 3) {
+        syslog(LOG_ERR, "Invalid number of arguements: %d", (argc - 1));
+        print_usage();
+        return 1;
+    }
+
+    char* writefile = argv[1];
+    char* writestr = argv[2];
```

Comment:
I like the idea.  See discussion about const correct for one improvement, discussed in class today.